### ISSUE TITLE
Does not build after utilizing rename command.

### ISSUE THREAD

**Issue by joshuaneedham** [nickjj/docker-rails-example#62]
Changed the name to customize for project start and ran into the following errors when running `docker compose up --build`. I did test with the stock repo version of this project without changing anything and it fired right up. The following output is displayed when I try to run the initial command after running the rename script.

At first I thought maybe it was the short 4 character name I chose but even after trying with a longer name it still failed.

You can find the repo [here](https://github.com/joshuaneedham/tmja/tree/initial-setup).

```track_my_job_apps-cable-1     | [1] Puma starting in cluster mode...
track_my_job_apps-cable-1     | [1] * Puma version: 6.2.2 (ruby 3.2.2-p53) ("Speaking of Now")
track_my_job_apps-cable-1     | [1] *  Min threads: 1
track_my_job_apps-cable-1     | [1] *  Max threads: 1
track_my_job_apps-cable-1     | [1] *  Environment: development
track_my_job_apps-cable-1     | [1] *   Master PID: 1
track_my_job_apps-cable-1     | [1] *      Workers: 1
track_my_job_apps-cable-1     | [1] *     Restarts: (✔) hot (✖) phased
track_my_job_apps-cable-1     | [1] * Preloading application
track_my_job_apps-cable-1     | [1] ! Unable to load application: SyntaxError: /app/config/application.rb:9: class/module name must be CONSTANT
track_my_job_apps-cable-1     |   class Application < Rails::Application
track_my_job_apps-cable-1     |        ^~~~~~~~~~~~~~~~~
track_my_job_apps-cable-1     | 
track_my_job_apps-cable-1     | /app/config/environment.rb:2:in `require_relative': /app/config/application.rb:9: class/module name must be CONSTANT (SyntaxError)
track_my_job_apps-cable-1     |   class Application < Rails::Application
track_my_job_apps-cable-1     |        ^~~~~~~~~~~~~~~~~
track_my_job_apps-cable-1     | 
track_my_job_apps-cable-1     |         from /app/config/environment.rb:2:in `<top (required)>'
track_my_job_apps-cable-1     |         from cable/config.ru:3:in `require_relative'
track_my_job_apps-cable-1     |         from cable/config.ru:3:in `block in <main>'
track_my_job_apps-cable-1     |         from /usr/local/bundle/gems/rack-2.2.7/lib/rack/builder.rb:116:in `eval'
track_my_job_apps-cable-1     |         from /usr/local/bundle/gems/rack-2.2.7/lib/rack/builder.rb:116:in `new_from_string'
track_my_job_apps-cable-1     |         from /usr/local/bundle/gems/rack-2.2.7/lib/rack/builder.rb:105:in `load_file'
track_my_job_apps-cable-1     |         from /usr/local/bundle/gems/rack-2.2.7/lib/rack/builder.rb:66:in `parse_file'
track_my_job_apps-cable-1     |         from /usr/local/bundle/gems/puma-6.2.2/lib/puma/configuration.rb:366:in `load_rackup'
track_my_job_apps-cable-1     |         from /usr/local/bundle/gems/puma-6.2.2/lib/puma/configuration.rb:288:in `app'
track_my_job_apps-cable-1     |         from /usr/local/bundle/gems/puma-6.2.2/lib/puma/runner.rb:158:in `load_and_bind'
track_my_job_apps-cable-1     |         from /usr/local/bundle/gems/puma-6.2.2/lib/puma/cluster.rb:359:in `run'
track_my_job_apps-cable-1     |         from /usr/local/bundle/gems/puma-6.2.2/lib/puma/launcher.rb:194:in `run'
track_my_job_apps-cable-1     |         from /usr/local/bundle/gems/puma-6.2.2/lib/puma/cli.rb:75:in `run'
track_my_job_apps-cable-1     |         from /usr/local/bundle/gems/puma-6.2.2/bin/puma:10:in `<top (required)>'
track_my_job_apps-cable-1     |         from /usr/local/bundle/bin/puma:25:in `load'
track_my_job_apps-cable-1     |         from /usr/local/bundle/bin/puma:25:in `<main>'
track_my_job_apps-cable-1 exited with code 1
track_my_job_apps-worker-1    | bundler: failed to load command: sidekiq (/usr/local/bundle/bin/sidekiq)
track_my_job_apps-worker-1    | /app/config/environment.rb:2:in `require_relative': /app/config/application.rb:9: class/module name must be CONSTANT (SyntaxError)
track_my_job_apps-worker-1    |   class Application < Rails::Application
track_my_job_apps-worker-1    |        ^~~~~~~~~~~~~~~~~
track_my_job_apps-worker-1    | 
track_my_job_apps-worker-1    |         from /app/config/environment.rb:2:in `<top (required)>'
track_my_job_apps-worker-1    |         from <internal:/usr/local/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:37:in `require'
track_my_job_apps-worker-1    |         from <internal:/usr/local/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:37:in `require'
track_my_job_apps-worker-1    |         from /usr/local/bundle/gems/sidekiq-7.1.0/lib/sidekiq/cli.rb:302:in `boot_application'
track_my_job_apps-worker-1    |         from /usr/local/bundle/gems/sidekiq-7.1.0/lib/sidekiq/cli.rb:42:in `run'
track_my_job_apps-worker-1    |         from /usr/local/bundle/gems/sidekiq-7.1.0/bin/sidekiq:31:in `<top (required)>'
track_my_job_apps-worker-1    |         from /usr/local/bundle/bin/sidekiq:25:in `load'
track_my_job_apps-worker-1    |         from /usr/local/bundle/bin/sidekiq:25:in `<top (required)>'
track_my_job_apps-worker-1    |         from /usr/local/lib/ruby/3.2.0/bundler/cli/exec.rb:58:in `load'
track_my_job_apps-worker-1    |         from /usr/local/lib/ruby/3.2.0/bundler/cli/exec.rb:58:in `kernel_load'
track_my_job_apps-worker-1    |         from /usr/local/lib/ruby/3.2.0/bundler/cli/exec.rb:23:in `run'
track_my_job_apps-worker-1    |         from /usr/local/lib/ruby/3.2.0/bundler/cli.rb:492:in `exec'
track_my_job_apps-worker-1    |         from /usr/local/lib/ruby/3.2.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
track_my_job_apps-worker-1    |         from /usr/local/lib/ruby/3.2.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
track_my_job_apps-worker-1    |         from /usr/local/lib/ruby/3.2.0/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
track_my_job_apps-worker-1    |         from /usr/local/lib/ruby/3.2.0/bundler/cli.rb:34:in `dispatch'
track_my_job_apps-worker-1    |         from /usr/local/lib/ruby/3.2.0/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
track_my_job_apps-worker-1    |         from /usr/local/lib/ruby/3.2.0/bundler/cli.rb:28:in `start'
track_my_job_apps-worker-1    |         from /usr/local/lib/ruby/gems/3.2.0/gems/bundler-2.4.10/libexec/bundle:45:in `block in <top (required)>'
track_my_job_apps-worker-1    |         from /usr/local/lib/ruby/3.2.0/bundler/friendly_errors.rb:117:in `with_friendly_errors'
track_my_job_apps-worker-1    |         from /usr/local/lib/ruby/gems/3.2.0/gems/bundler-2.4.10/libexec/bundle:33:in `<top (required)>'
track_my_job_apps-worker-1    |         from /usr/local/bin/bundle:25:in `load'
track_my_job_apps-worker-1    |         from /usr/local/bin/bundle:25:in `<main>'
track_my_job_apps-worker-1 exited with code 1
track_my_job_apps-web-1       | <internal:/usr/local/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:37:in `require': /app/config/application.rb:9: class/module name must be CONSTANT (SyntaxError)
track_my_job_apps-web-1       |   class Application < Rails::Application
track_my_job_apps-web-1       |        ^~~~~~~~~~~~~~~~~
track_my_job_apps-web-1       | 
track_my_job_apps-web-1       |         from <internal:/usr/local/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:37:in `require'
track_my_job_apps-web-1       |         from /usr/local/bundle/gems/bootsnap-1.16.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
track_my_job_apps-web-1       |         from /usr/local/bundle/gems/railties-7.0.4.3/lib/rails/commands/server/server_command.rb:137:in `block in perform'
track_my_job_apps-web-1       |         from <internal:kernel>:90:in `tap'
track_my_job_apps-web-1       |         from /usr/local/bundle/gems/railties-7.0.4.3/lib/rails/commands/server/server_command.rb:134:in `perform'
track_my_job_apps-web-1       |         from /usr/local/bundle/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
track_my_job_apps-web-1       |         from /usr/local/bundle/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
track_my_job_apps-web-1       |         from /usr/local/bundle/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
track_my_job_apps-web-1       |         from /usr/local/bundle/gems/railties-7.0.4.3/lib/rails/command/base.rb:87:in `perform'
track_my_job_apps-web-1       |         from /usr/local/bundle/gems/railties-7.0.4.3/lib/rails/command.rb:48:in `invoke'
track_my_job_apps-web-1       |         from /usr/local/bundle/gems/railties-7.0.4.3/lib/rails/commands.rb:18:in `<main>'
track_my_job_apps-web-1       |         from <internal:/usr/local/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:37:in `require'
track_my_job_apps-web-1       |         from <internal:/usr/local/lib/ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:37:in `require'
track_my_job_apps-web-1       |         from /usr/local/bundle/gems/bootsnap-1.16.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
track_my_job_apps-web-1       |         from bin/rails:4:in `<main>'
track_my_job_apps-web-1 exited with code 1```

**Comment by nickjj** [comment]
Hi,

Your repo's link isn't coming up, is it a private repo?

Did you try running a `docker compose down -v` before upping the project after you renamed it?

**Comment by joshuaneedham** [comment]
Yes, my apologies for not switching it before linking. Should be public now.

**Comment by joshuaneedham** [comment]
> Hi,
> 
> Your repo's link isn't coming up, is it a private repo?
> 
> Did you try running a `docker compose down -v` before upping the project after you renamed it?

I just did that and the same got the same result.

**Comment by joshuaneedham** [comment]
I figured it out. It didn't like the `module track_my_job_apps` in the `application.rb file. When I ran the command I must have input it the wrong way for the script to properly update the application name. I manually updated the file with `module TrackMyJobApps`, shut everything down, and then rebuilt. Works like a charm.

Thanks again for this awesome project. I can't wait to work my existing work in to this framework.
